_distro_map.yml

```diff
@@ -31,4 +31,7 @@ openshift-enterprise:
       dir: enterprise/latest
     enterprise-2.2:
       name: Version 2.2
-      dir: enterprise/v2.2
+      dir: enterprise/2.2
+    enterprise-3.0:
+      name: Version 3.0
```

Comment:
@nhr For brevity's sake, what do you think about dropping "Version" from the two names here (for 2.2 and 3.0)? Mostly considering the topic title here[1] and the breadcrumb.

[1] http://docs.openshift.com/enterprise/3.0/welcome/index.html